content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -231,184 +235,185 @@ and in the logs to monitor the state of the auditing subsystem.
+   # as a sidecar container.
+   <source>
+       @type tail
+       # audit log path of kube-apiserver
+       path /var/log/kube-audit
+       pos_file /var/log/audit.pos
+       format json
+       time_key time
+       time_format %Y-%m-%dT%H:%M:%S.%N%z
+       tag audit
+   </source>
+
+   <filter audit>
+       #https://github.com/fluent/fluent-plugin-rewrite-tag-filter/issues/13
        @type record_transformer
-       remove_keys namespace
-    </filter>
-
-    <match audit.**>
-        @type forest
-        subtype file
-        remove_prefix audit
-        <template>
-            time_slice_format %Y%m%d%H
-            compress gz
-            path /var/log/audit-${tag}.*.log
-            format json
-            include_time_key true
-        </template>
-    </match>
-    EOF
-    ```
+       enable_ruby
+       <record>
+        namespace ${record["objectRef"].nil? ? "none":(record["objectRef"]["namespace"].nil? ? "none":record["objectRef"]["namespace"])}
+       </record>
+   </filter>
+
+   <match audit>
+       # route audit according to namespace element in context
+       @type rewrite_tag_filter
+       <rule>
+           key namespace
+           pattern /^(.+)/
+           tag ${tag}.$1
+       </rule>
+   </match>
+
+   <filter audit.**>
+      @type record_transformer
+      remove_keys namespace
+   </filter>
+
+   <match audit.**>
+       @type forest
+       subtype file
+       remove_prefix audit
+       <template>
+           time_slice_format %Y%m%d%H
+           compress gz
+           path /var/log/audit-${tag}.*.log
+           format json
+           include_time_key true
+       </template>
+   </match>
+   ```
 
 1. Start fluentd
 
-    ```shell
-    fluentd -c /etc/fluentd/config  -vv
-    ```
+   ```shell
+   fluentd -c /etc/fluentd/config -vv
+   ```
 
-1. Start kube-apiserver with the following options:
+1. Start `kube-apiserver` with the following options:
 
-    ```shell
-    --audit-policy-file=/etc/kubernetes/audit-policy.yaml --audit-log-path=/var/log/kube-audit --audit-log-format=json
-    ```
+   ```shell
+   kube-apiserver ... --audit-policy-file=/etc/kubernetes/audit-policy.yaml --audit-log-path=/var/log/kube-audit --audit-log-format=json
+   ```
 
 1. Check audits for different namespaces in `/var/log/audit-*.log`
 
-### Use logstash to collect and distribute audit events from webhook backend
+### Use Logstash to collect and distribute audit events from webhook backend
 
 [Logstash](https://www.elastic.co/products/logstash)
 is an open source, server-side data processing tool. In this example,
 we will use logstash to collect audit events from webhook backend, and save events of
 different users into different files.
 
-1. install [logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html)
+1. install [Logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html)
```

Comment:
nit: Is this page a candidate for the third-party-content label?
nit: Capitalize: Install, Create, Start, Check?

Reply:
> is this page a candidate for the third-party-content label?

Probably (and I would track that as a separate issue)

Reply:
Has an issue been logged?